### PR TITLE
[CI] Extend nightly tests on both v6e and v7x

### PR DESCRIPTION
**Pull request by meiyeh123**
# Description

- This PR introduces placeholders into the YML files to enable the use of different variables for both v6e and v7x during the upload of nightly CI procedures. This applies to `main.yml`.

- It also use group to make buildkite UI looks better

- Organizing output CSV files into dedicated 'v6/' and 'v7/' directories.

# Tests

[buildkite](https://buildkite.com/tpu-commons/tpu-inference-ci/builds/9685/steps/canvas)

# Checklist

Before submitting this PR, please make sure:
- I have performed a self-review of my code.
- I have necessary comments in my code, particularly in hard-to-understand areas.
- I have made or will make corresponding changes to any relevant documentation.

### CODE REVIEW

**Review by CienetStingLin — APPROVED** [review]

**Review by QiliangCui — APPROVED** [review]

File: .buildkite/main.yml [hunk 1]
```diff
@@ -42,18 +42,19 @@ steps:
...
-     <<: *run-on-nightly-or-tag
+    #  <<: *run-on-nightly-or-tag
      key: commit_support_matrices
      depends_on: generate_support_matrices
      agents:
        queue: cpu
-     secrets:
-      - GITHUB_PAT
+    #  secrets:
+    #   - GITHUB_PAT
```
Comment (jcyang43) [inline .buildkite/main.yml]: I don't think changes in this file are meant to be checked in?

File: .buildkite/parallelism/DP.yml [hunk 2]
```diff
@@ -15,51 +15,53 @@
...
+      queue: "${TPU_QUEUE_MULTI:-tpu_v6e_8_queue}"
     commands:
       - |
         .buildkite/scripts/run_in_docker.sh \
-          bash -c 'python3 -m pytest -s -v -x /workspace/tpu_inference/tests/e2e/test_data_parallel.py::test_dp_correctness'
-  - label: "Record correctness test result for DP"
-    key: "record_DP_CorrectnessTest"
-    depends_on: "DP_CorrectnessTest"
+          bash -c 'python3 -m pytest -s -v -x /workspace/tpu_inference/tests/e2e/test_data_parallel.py'
+  - label: "${LABEL_PREFIX}Record correctness test result for DP"
```
Comment (jcyang43) [inline .buildkite/parallelism/DP.yml]: It looks like `LABEL_PREFIX` has been removed in `bootstrap.sh`? 

Image

File: .buildkite/features/async_scheduler.yml [hunk 3]
```diff
@@ -15,43 +15,49 @@
 # pipeline-name: async scheduler
 # pipeline-type: feature support matrix
 steps:
-  - label: "Correctness tests for async scheduler"
-    key: "async_scheduler_CorrectnessTest"
+  - label: "${TPU_VERSION}Correctness tests for async scheduler"
+    key: "${KEY_PREFIX}async_scheduler_CorrectnessTest"
```
Comment (jcyang43) [inline .buildkite/features/async_scheduler.yml]: Just a general question: what's the difference between `TPU_VERSION` & `KEY_PREFIX`? Do we need both? 

If we only need one of them, I'd suggest keeping `TPU_VERSION` as the name is more descriptive. Thanks

Comment (boe20211) [reply .buildkite/features/async_scheduler.yml]: TPU_VERSION="TPU7x "
Used for display labels in the Buildkite UI. It includes a trailing space to ensure the UI labels are formatted correctly and readable.

Example: 'TPU7x Correctness tests for EP'

KEY_PREFIX="tpu7x_"
Used for Buildkite step keys and internal logic.Spaces are strictly prohibited in Buildkite keys. Using underscores ensures that step dependencies and metadata retrieval remain stable.

Example: "tpu7x_DP_CorrectnessTest"

File: .buildkite/parallelism/DP.yml (hunk 2 above)
Comment (boe20211) [reply .buildkite/parallelism/DP.yml]: Thanks for catching that! fixed

File: .buildkite/features/async_scheduler.yml (hunk 3 above)
Comment (boe20211) [reply .buildkite/features/async_scheduler.yml]: Resolved: Replaced KEY_PREFIX with TPU_VERSION.

File: .buildkite/main.yml
Comment (meiyeh123) [inline .buildkite/main.yml]: the key tpu_test_notification is outdated

- _tpu_test_notification
- tpu7x_tpu_test_notification

Comment (boe20211) [reply .buildkite/main.yml]: fixed, thanks Dennis.

File: .buildkite/main.yml [hunk 4]
```diff
@@ -66,7 +87,8 @@ steps:
      key: record_verified_commit_hashes
      depends_on:
        - notify_test_results
-       - tpu_test_notification
+       - _tpu_test_notification
```
Comment (meiyeh123) [inline .buildkite/main.yml]: Nit: Could we explicitly export ${TPU_VERSION} for v6e instead of leaving it blank? It would improve readability.
For example, export TPU_VERSION="tpu6e" or replace all labels with ${TPU_VERSION:-tpu6e}

Comment (meiyeh123) [reply .buildkite/main.yml]: For both pipeline_jax.yml and main.yml steps

Comment (boe20211) [reply .buildkite/main.yml]: done
